xarray/tests/test_conventions.py

```diff
@@ -156,12 +156,16 @@ def test(self):
 
 
 def test_decode_cf_with_conflicting_fill_missing_value():
-    var = Variable(['t'], np.arange(10),
+    expected = Variable(['t'], [np.nan, np.nan, 2], {'units': 'foobar'})
+    expected[[0, 1]] = np.nan
```

Comment:
I think this line is unnecessary, since you already set these values to NaN when you construct the variable.

Reply:
Oops, leftover from trying to get the thing to work. 😁